tpdatasrc/tpgamefiles/scr/feats/feat - Craven.py

```diff
@@ -6,4 +6,6 @@ def CheckPrereq(attachee, classLevelled, abilityScoreRaised):
 	#Sneak Attack Check
 	if not attachee.has_feat(feat_sneak_attack):
 		return 0
+	if attachee.stat_level_get(stat_level_paladin) > 1: #workaround until I figure out how to check for immunity to fear

```

Comment:
There's a dispatch for immunity checks. I think it's not yet exposed to Python, but you may wish to extend it.